app/components/UI/Modal.js

```diff
@@ -28,6 +28,27 @@ class Modal extends React.Component {
     withHeader: false
   }
 
+  constructor(props) {
```

Comment:
This `constructor` method can be removed if you change the `closeOnEscape` method to be a fat arrow function:

```js
  closeOnEscape(e) {
    if (e.key === 'Escape') {
      const { onClose } = this.props

       onClose()
    }
  }
```
becomes: 
```js
  closeOnEscape = e => {
    if (e.key === 'Escape') {
      const { onClose } = this.props

      onClose()
    }
  }
```

Reply:
Sure, I can do that.